.github/workflows/ruby_build.yml

```diff
@@ -0,0 +1,33 @@
+name: Ruby Build
+
+on:
+  release:
+    types: published
```

Comment:
does this exclude pre-release? So would it work for UAT builds marked pre-release?

Reply:
It includes pre-releases, just not draft releases.

Reply:
Ah OK no probs, I hadn't really noticed drafts were a thing